### PR TITLE
Move more inspector functions to `InspectorShared.js`

### PR DESCRIPTION
**Pull request by lilleyse**
`Cesium3DTilesInspector` and `VoxelInspector` had their own copies of `makeRangeInput` and `makeButton`.

This PR moves that functionality into a common location in `InspectorShared.js`.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/inspector-cleanup/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: packages/widgets/Source/InspectorShared.js [hunk 1]
```diff
@@ -83,4 +83,82 @@ InspectorShared.createSection = function (
...
+ * @return {Element}
+ */
+InspectorShared.createButton = function (
+  buttonText,
+  clickedBinding,
+  activeBinding
+) {
+  const button = document.createElement("button");
+  button.type = "button";
+  button.textContent = buttonText;
```
Comment (ggetz) [inline packages/widgets/Source/InspectorShared.js:153]: Like the above function, the required parameters should be checked.

**Review by ggetz — COMMENTED** [review]
Thanks @lilleyse! This looks good other than that one comment.

**Comment by ggetz** [comment]
Thanks @lilleyse!

**Pull request merged**